services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -218,11 +158,9 @@ def raise_if_blocked(
             The list of blocked datasets. If empty, no dataset is blocked.
     Returns:
         `None`
-    <Tip>
     Raises the following errors:
         - [`~job_runners.config.parquet_and_info.DatasetInBlockListError`]
```

Comment:
```suggestion
        - [`~libcommon.exceptions.DatasetInBlockListError`]
```

Reply:
You're right. I fix a small detail (no `~` prefix):

```suggestion
        - [`libcommon.exceptions.DatasetInBlockListError`]
```